homeassistant/components/dwd_weather_warnings/const.py

```diff
@@ -29,5 +34,8 @@
 CURRENT_WARNING_SENSOR: Final = "current_warning_level"
 ADVANCE_WARNING_SENSOR: Final = "advance_warning_level"
 
-DEFAULT_NAME: Final = "DWD-Weather-Warnings"
+DEFAULT_NAME: Final = "DWD Weather Warnings"
 DEFAULT_SCAN_INTERVAL: Final = timedelta(minutes=15)
+DEFAULT_MONITORED_CONDITIONS: Final = [CURRENT_WARNING_SENSOR, ADVANCE_WARNING_SENSOR]
```

Comment:
If this is only used on the old YAML, I think you should keep it in `sensor.py`.
Maybe rename it as `YAML_MONITORED_CONDITIONS` with a comment to say it can be dropped when YAML goes away.